<launch>
  <!-- ************** Global Parameters ***************  -->
  <param name="/use_sim_time" value="true"/>
  <node pkg="cv_lidar_fusion_pkg" type="random_obstacles.py" name="random_obstacles" />
</launch>